File: php/src/FinalSol/HourReporter.php
<?php

declare(strict_types=1);

namespace App\FinalSol;

class HourReporter
{
    /**
     * @var WorkHours
     */
    private $work_hours;

    public function __construct(WorkHours $work_hours)
    {
        $this->work_hours = $work_hours;
    }

    public function reportHours(): string
    {
        return sprintf('Regular Hours: %s', $this->work_hours->regularHours());
    }
}

File: java/src/main/java/community/dddtw/refactor/complete/HourReporter.java
package community.dddtw.refactor.complete;

public class HourReporter {
    private WorkingWeek workingWeek;

    public HourReporter(WorkingWeek workingWeek) {
        this.workingWeek = workingWeek;
    }

    public String reportHours() {
        return "Regular Hours: " + this.workingWeek.regularHours();
    }
}
